custom/icds_reports/reports/disha.py

```diff
@@ -64,51 +63,37 @@ def _get_columns(self):
         columns.remove("month")
         return columns
 
-    def _get_rows(self):
-        return DishaIndicatorView.objects.filter(
+    def _get_rows(self, query_master=False):
+        if query_master:
+            objects = DishaIndicatorView.objects.using('icds-ucr')
```

Comment:
@emord FYI. We discussed this a while ago where we had to retry queries because postgres standby was cancelling queries. It still fails sometimes, so I am forcing to query master after max retires on standby.

Reply:
given the frequency that it failed on the standby, should you just always query master?

Reply:
It succeeds more than it fails, so worth trying standby first. It's a background task anyhow.